rclpy/rclpy/client.py

```diff
@@ -64,6 +64,8 @@ def __init__(
         # True when the callback is ready to fire but has not been "taken" by an executor
         self._executor_event = False
 
+        self.call_lock = threading.Lock()
```

Comment:
simplify the lock name for class weak internal used just like others?

```suggestion

        self._lock = threading.Lock()
```

Reply:
Done